python/delta_sharing/delta_sharing.py

```diff
@@ -73,17 +74,24 @@ def load_as_pandas(
         rest_client=DataSharingRestClient(profile),
         limit=limit,
         version=version,
+        timestamp=timestamp
     ).to_pandas()
 
 
-def load_as_spark(url: str, version: Optional[int] = None) -> "PySparkDataFrame":  # noqa: F821
+def load_as_spark(
+    url: str,
+    version: Optional[int] = None,
+    timestamp: Optional[str] = None
+) -> "PySparkDataFrame":  # noqa: F821
     """
     Load the shared table using the given url as a Spark DataFrame. `PySpark` must be installed,
     and the application must be a PySpark application with the Apache Spark Connector for Delta
     Sharing installed.
 
     :param url: a url under the format "<profile>#<share>.<schema>.<table>".
     :param version: an optional non-negative int. Load the snapshot of table at version.
+    :param timestamp: an optional string. Load the snapshot of table at version corresponding
+      to the timestamp.
```

Comment:
Should we add comment that at most one of version/timestamp should be specified?

Reply:
added.